frontend/javascripts/oxalis/view/context_menu.tsx

```diff
@@ -422,7 +423,8 @@ function getMeshItems(
             !isProofreadingActive ||
             activeSegmentMissing ||
             clickedMeshId === activeCellId ||
-            maybeUnmappedSegmentId == null,
+            maybeUnmappedSegmentId == null ||
+            meshFileMapping != null,
```

Comment:
:thinking: There is quite some code duplication here in the boolean clause. While you are already editing this file, could you maybe please refactor the boolean expression together with the one in ll 456 ff.? E.g. call it `const proofreadingActionsDisabled = <shared_boolean_expression>` or so :thinking:

Reply:
And while I am at it: The conditional error messages are also pretty much duplicated (ll 439 ff. and ll 475 ff.) Could you maybe extract shared things here too? That would be very awesome :t-rex: